dashboard/app/transactions/page.tsx

```
@@ -33,10 +34,134 @@ const getRiskLevel = (amount: string) => {
+    }));
+  };
+
+  const hasActiveFilters = 
+    filters.dateFrom || 
+    filters.dateTo || 
+    filters.amountMin || 
+    filters.amountMax || 
+    filters.status.length > 0 || 
+    filters.riskLevel.length > 0;
+
+  const filteredTransactions = transactions.filter((transaction) => {
```

Biome (2.3.13)

[error] Shouldn't redeclare 'filteredTransactions'. Consider to delete it or rename it.

'filteredTransactions' is defined here:


(lint/suspicious/noRedeclare)